index.ts

```diff
@@ -2,6 +2,8 @@ import { Application, ProbotOctokit } from "probot";
 import { OctokitResponse, PullsListFilesResponseData } from "@octokit/types";
 type Octokit = InstanceType<typeof ProbotOctokit>;
 
+const NEW_PLUGIN = "new plugin";
+const REMOVE_PLUGIN = "remove plugin";
```

Comment:
Maybe "plugin (added|removed)"?

Reply:
Good idea, would fit with the existing `plugin change`